cfpb/sbl-frontend
# Upgrade to Node 22 (2nd attempt)

Closes #1040 

## Changes
- Updated Dockerfile to use new `ghcr.io/cfpb/regtech/sbl/node-js-alpine:3.20` (node 22) base image.  
- Updated `package.json -> engines` to use Node v22
- Updated `.nvmrc` to use `v22`


## How to test this PR

1. Pull branch
2. `nvm use`
3. `yarn start` to easily launch all backend services
4. `yarn docker-build-and-run` to run the container version of the Frontend
5. Update your `.env` file to run tests against the container version
   ```
   SBL_PLAYWRIGHT_TEST_TARGET="http://localhost:8085"
   ```
6. `yarn playwright test --workers 4`
7. Verify all tests pass
